src/pages/Environments.js

```diff
@@ -463,7 +463,8 @@ const Environments = () => {
   }
 
   const renderWorkspaceForRuntimes = runtime => {
-    const { status, googleProject, labels: { saturnWorkspaceNamespace = googleProject, saturnWorkspaceName } } = runtime
+    const { status, googleProject, labels: { saturnWorkspaceNamespace = googleProject, saturnWorkspaceName } = {} } = runtime
```

Comment:
will this mean if `labels` doesn't exist, then both `saturnWorkspaceNamespace`, `saturnWorkspaceName` will be undefined?

I think what happens in many places is that only `saturnWorkspaceNamespace` doesn't exist but `labels` field does still exist?

Reply:
In both cases (saturnWorkspaceNamespace being undefined and labels being undefined), saturnWorkspaceNamespace will resolve to undefined. This is just a guard in case it ever is undefined